backend/schedulerd/executor.go

```diff
@@ -233,7 +238,7 @@ func (a *AdhocRequestExecutor) publishProxyCheckRequests(entities []*types.Entit
 }
 
 func (a *AdhocRequestExecutor) execute(check *types.CheckConfig) error {
-	request := a.buildRequest(check)
+	request, _ := a.buildRequest(check)
```

Comment:
Would it be beneficial to catch and return the error here from `buildRequest`?

Reply:
In this instance, nil is always returned by AdhocRequestExecutor.buildRequest.

Reply:
Hopefully if that changes in the future, we remember to catch it.. I don't foresee that happening though 😉